contribs/drt-extensions/src/test/java/org/matsim/contrib/drt/extension/fiss/RunFissDrtScenarioIT.java

```diff
@@ -146,18 +149,43 @@ public void test() {
 		final Controler run = DrtOperationsControlerCreator.createControler(config, false);
 
 		//FISS part
+		LinkCounter linkCounter = new LinkCounter();
 		{
 			FISSConfigGroup fissConfigGroup = ConfigUtils.addOrGetModule(config, FISSConfigGroup.class);
 			fissConfigGroup.sampleFactor = 0.1;
 			fissConfigGroup.sampledModes = Set.of(TransportMode.car);
+			fissConfigGroup.switchOffFISSLastIteration = true;
 			FISSConfigurator.configure(run);
 
 			QSimComponentsConfigurator qSimComponentsConfigurator = FISSConfigurator
 					.activateModes(List.of(), MultiModeDrtConfigGroup.get(config).modes().collect(Collectors.toList()));
 
 			run.configureQSimComponents(qSimComponentsConfigurator);
+
+			run.addOverridingModule(new AbstractModule() {
+				@Override
+				public void install() {
+					addEventHandlerBinding().toInstance(linkCounter);
+				}
+			});
+
 		}
 
 		run.run();
+		assert(linkCounter.getLinkLeaveCount()==23961);
```

Comment:
`assert` is a java-keyword, which musst explicitly be enabled to have any effect (`--enable-assertions`, `-ea`). Maven enables this when running tests, so it should be fine. But I would still propose using JUnit's assert-methods instead (`Assert.assertEquals(23961, linkCounter.getLinkLeaveCount())`) as this will provide the better error message when it fails.

Reply:
Thanks, done!